libs/langchain-google-common/README.md

```diff
@@ -43,5 +43,10 @@ Tasks and services still to be implemented:
 * Multimodal embeddings
 * Vertex AI Search
 * Vertex AI Model Garden
+  * Online prediction endpoints
```

Comment:
Should also update table here:

https://js.langchain.com/docs/integrations/chat/